sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/SqlTransform.java

```diff
@@ -298,6 +304,11 @@ public SqlTransform registerUdaf(String functionName, Combine.CombineFn combineF
     return toBuilder().setUdafDefinitions(newUdafs).build();
   }
 
+  public SqlTransform withErrorsTransformer(
+      PTransform<PCollection<BeamCalcRelError>, POutput> errorsTransformer) {
```

Comment:
Can you make this `PTransform<PCollection<BeamCalcRelError>, ? extends POutput>` instead? (Or really `PTransform<PCollection<Row>, ? extends POutput>`)

Reply:
sure thing: https://github.com/apache/beam/pull/15915/commits/c0abbcee7f3c34e631599cc44007dacceece3e46